int8 speed
string cmd_msg
---
string resp_msg
